src/main/java/org/broadinstitute/consent/http/resources/DaaResource.java

```diff
@@ -77,4 +78,15 @@ public Response createDaaForDac(
     }
   }
 
+  @GET
+  @Produces(MediaType.APPLICATION_JSON)
+  @RolesAllowed({ADMIN, MEMBER, CHAIRPERSON, RESEARCHER})
```

Comment:
This could be `@PermitAll` - I don't see any reason to restrict the access here. SO's will definitely need to see them as well.